src/components/Header/index.js

```diff
@@ -1,24 +1,49 @@
 import React, { Component } from 'react';
-import { Button } from 'react-bootstrap';
 import './index.css';
+import logo from './img/logo.png';
 
 export class Header extends Component {
+  constructor() {
+    super()
```

Comment:
Why did you use constructor () to define state?
If it not necessary please do it like in other components.